src/v/pandaproxy/rest/configuration.h

```diff
@@ -26,13 +27,15 @@ namespace pandaproxy::rest {
 /// All application modules depend on configuration. The configuration module
 /// can not depend on any other module to prevent cyclic dependencies.
 struct configuration final : public config::config_store {
-    config::one_or_many_property<model::broker_endpoint> pandaproxy_api;
+    config::one_or_many_property<config::rest_authn_endpoint> pandaproxy_api;
     config::one_or_many_property<config::endpoint_tls_config>
       pandaproxy_api_tls;
     config::one_or_many_property<model::broker_endpoint>
       advertised_pandaproxy_api;
     config::property<ss::sstring> api_doc_dir;
     config::property<std::chrono::milliseconds> consumer_instance_timeout;
+    config::property<int64_t> client_cache_max_size;
+    config::property<int64_t> client_keep_alive;
```

Comment:
Why not a stronger type like `std::chrono::milliseconds`?

Reply:
Fair point. I was originally using `model::timestamp::type` which is an alias for `int64_t`. Will change.